src/server/game/Globals/ObjectMgr.cpp

```diff
@@ -9352,6 +9352,33 @@ void ObjectMgr::LoadFactionChangeQuests()
     LOG_INFO("server.loading", " ");
 }
 
+void ObjectMgr::LoadInstanceSavedGameobjectStateData()
```

Comment:
should probably move this to where gameobjects are loaded or somewhere better and not mid FactionChange loads :D